fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -183,6 +188,11 @@ def self.available_options
                                        env_name: "FL_CARTHAGE_LOG_PATH",
                                        description: "Path to the xcode build output",
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :archive,
+                                       env_name: "FL_CARTHAGE_ARCHIVE",
+                                       description: "Archive built frameworks from the current project",
+                                       is_string: false,
+                                       default_value: false),
```

Comment:
I think you should add `type: Boolean` here as same as other arguments.

```suggestion
          FastlaneCore::ConfigItem.new(key: :archive,
                                       env_name: "FL_CARTHAGE_ARCHIVE",
                                       description: "Archive built frameworks from the current project",
                                       is_string: false,
                                       type: Boolean,
                                       default_value: false),
```

Reply:
Thanks for your review!